package.json

```diff
@@ -14,6 +14,7 @@
     "release": "pnpm run build:release && changeset publish",
     "new-llamaindex": "pnpm run build:release && changeset version --ignore create-llama",
     "new-create-llama": "pnpm run build:release && changeset version --ignore llamaindex --ignore @llamaindex/core-test",
+    "new-experimental": "pnpm run build:release && changeset version --ignore create-llama",
```

Comment:
@EmanuelCampos this operation is then the same as `new-llamaindex`, you probably want to ignore more or leave it out (which would mean experimental versioning would depend on core)

Reply:
oh true hehe, mb